### ISSUE TITLE
Use database instead of filesystem

### ISSUE THREAD

**Issue by marblestation** [adsabs/ADSfulltext#57]
Extracted fulltext and metadata should be stored in a database (instead of in million of files in the NAS) to speed up access.

**Comment by spacemansteve** [comment]
Opinions may differ but I'd give this task a lower priority and would schedule it for a sprint after the new pipeline has been in production for a while.  I have nothing against moving the state information from the filesystem to the database.  I'd just like to have all the components reliably running in production before we start changing any components.

**Comment by romanchyla** [comment]
i concur; and i have some opinions about how the fulltext pipeline should change, let's talk before you decide to work on it
